coda-prototype/UIUtils.js

```diff
@@ -491,7 +490,55 @@ var UIUtils = (function() {
         isScrolledToTop: function(tableContainer) {
             return UIUtils.checkVisible($(".message-row").first()[0]);
             //return $("#message-panel").scrollTop() === 0;
-        }
+        },
+
+        /**
+         * Displays the given message in a green banner at the top of the screen.
+         * Success banners are automatically dismissed after a few seconds, and the contents automatically cleared.
+         * @param message HTML to display in the banner.
+         */
+        displayAlertAsSuccess(message) {
```

Comment:
Consider "displaySuccessAlert"

displayAlertAsSuccess sounds more like something that could be true or false, than a verb

Reply:
Hmm... The reason for this name was to stress that there is only one alert. When this alert is displayed, it can either be done so looking like success or failure.

Reply:
In any case, assuming we do #83 this is not important.